discord.html

```diff
@@ -8,21 +8,25 @@
   <link rel="stylesheet" href="./css/pages/discord.css">
 </head>
 <body>
-<h1 class="main-title">New to Real Dev Squad? Don't worry we are here to help you😊</h1>
+<h1 class="welcome-title">Real Dev Squad Server on Discord</h1>
+<br>
+<p class="main-heading">New to Real Dev Squad? Don't worry we are here to help you.</p>
 <p Class="discord-info" ><a href="https://discord.com/new">Discord</a> is a chat app, similar to apps such as Skype, or app widely used for professional communications like Slack. </p>
-<p Class="discord-info">It’s geared towards multiple users, providing them with ways to find each other, coordinate and talk while using it. It supports video calls, voice chat, and text, allowing users to get in touch however they please.</p>
-<p Class="discord-info">We have created different <em><strong>channels</strong></em> based on the task making things much easier to keep information straight & segregated.</p>
+<p Class="discord-info">It’s geared towards multiple users, providing them with ways to find each other, coordinate and talk while using it.</p>
+<p class="discord-info">It supports video calls, voice chat, and text, allowing users to get in touch however they please.</p>
+<p Class="discord-info">We have created different   <strong>channels</strong>   based on the task making things much easier to keep information straight & segregated.</p>
```

Comment:
instead of repeating class "discord-info" on every P tag; you can simply wrap all the P tags in a DIV and assign a "discord-info" to that. So, ur CSS will look like :
.discord-info > p {....}